tests/jvm/src/test/scala/scala/meta/tests/semanticdb/MetacScalaLibrary.scala

```diff
@@ -0,0 +1,46 @@
+package scala.meta.tests.semanticdb
+
+import java.io.File
+import java.net.URLClassLoader
+import java.nio.file.Files
+import java.nio.file.Paths
+import scala.meta.cli.Metac
+import scala.meta.tests.BuildInfo
+import org.langmeta.internal.io.FileIO
+import org.langmeta.io.AbsolutePath
+
+// Compile all of scala-library with metac and report any semanticdb errors.
+object MetacScalaLibrary {
+  def main(args: Array[String]): Unit = {
+    val library = Paths
+      .get("target")
+      .resolve("scala-library")
+      .resolve(s"scala-${BuildInfo.scalaVersion}")
+      .resolve("src")
+      .resolve("library")
+      .toAbsolutePath
+    assert(Files.isDirectory(library), s"$library is not a directory! Run `sbt ci-metac`")
+    val classpath = this.getClass.getClassLoader
+      .asInstanceOf[URLClassLoader]
+      .getURLs
+      .map(_.getPath)
+      .filter(_.contains("scala-library"))
+      .mkString(File.pathSeparator)
+    val files = FileIO
+      .listAllFilesRecursively(AbsolutePath(library))
+      .map(_.toString)
+      .filter(_.endsWith("scala"))
+    val out = Files.createTempDirectory("metac")
+    println(s"Compiling ${files.length} sources from scala-library...")
+    Metac.main(
+      Array(
+        "-d",
+        out.toString,
+        "-classpath",
+        classpath,
+        "-P:semanticdb:failures:error"
+      ) ++ files
+    )
+    println(out)
```

Comment:
Is this supposed to go into the stdout of our CI?

Reply:
It's helpful to know the directory to check out the generated semanticdb files when running this locally. It doesn't matter much for our CI